app/src/pages/metamine/visualizationNU/ScatterPlot.vue

```diff
@@ -3,19 +3,38 @@
     <template #main_chart>
       <Scatter />
       <div class="tools-simulation u--layout-flex u--layout-flex-justify-sb">
-        <button @click="toggleDialogBoxKnn()" class="nuplot-button"> Find Nearest Neighbors</button>
-        <dialog-box :active="dialogBoxActiveKnn" :disableClose="true">
```

Comment:
Because you are using a custom :active sync, I recommend keeping the `:disableClose="true"` line that was removed here back into line 15. Otherwise, the `x` button will continue to show up on the dialog box without being usable

Reply:
Sure, updated